modules/Bio/EnsEMBL/Analysis/Hive/RunnableDB/HiveAddPlaceholderLocation.pm

```diff
@@ -64,29 +73,49 @@ sub run {
 
   my $core_dba = $self->hrdb_get_con('input_db');
   my $sa = $core_dba->get_SliceAdaptor;
-
   my $sth_longest = $core_dba->dbc->prepare('select seq_region_id from seq_region order by length desc limit 10');
   $sth_longest->execute;
-
   my $sample_transcript;
- LOOP: while (my $seq_region_id = $sth_longest->fetchrow_array) {
-    my $region = $sa->fetch_by_seq_region_id($seq_region_id);
-
-  TRANSCRIPT:foreach my $transcript (@{ $region->get_all_Transcripts_by_type('protein_coding') }){
-      my $supporting_features = $transcript->get_all_supporting_features;
-      foreach my $support (@$supporting_features){
-        if ($support->hcoverage() >= 99 && $support->percent_id() >= 75){
-          $sample_transcript=$transcript;
-          last LOOP;
+
+  if ($self->{'has_supporting'} == 1) {
+
+    LOOP: while (my $seq_region_id = $sth_longest->fetchrow_array) {
+      my $region = $sa->fetch_by_seq_region_id($seq_region_id);
+      TRANSCRIPT:foreach my $transcript (@{ $region->get_all_Transcripts_by_type('protein_coding') }) {
+        my $supporting_features = $transcript->get_all_supporting_features;
+        foreach my $support (@$supporting_features) {
+          print $support->hcoverage(),"\t",$support->percent_id(),"\n";
```

Comment:
Could you please use $self->say_with_header(<message>) to print the message?

Reply:
Removed instead, that was just a diagnistic print, doesn't need to be stored.